src/vs/platform/terminal/node/terminalProfiles.ts

```diff
@@ -190,7 +196,7 @@ async function initializeWindowsProfiles(): Promise<void> {
 	profileSources.set('PowerShell', {
 		profileName: 'PowerShell',
 		paths: await getPowershellPaths(),
-		icon: 'terminal-powershell'
+		icon: { id: 'terminal-powershell' }
```

Comment:
For all these icon references here it would be good to reference `Codicon.terminal*`. I see you can't just set `icon: `Codicon.terminalPowershell` here as it's a class that we need to transfer over IPC, a helper method like this would be useful to reduce repetition:

```ts
namespace ThemeIcon {
  asThemeIcon(codicon: Codicon) {
    return { id: codicon.id };
  }
}
```